tests/test_utils/test_schemaloader.py

```diff
@@ -39,6 +39,11 @@ def test_imports(input_path, snapshot):
     assert loader.synopsis.errors() == []
 
 
+def test_imports_relative(input_path):
+    loader = SchemaLoader(input_path("relative_import_test/main.yaml"))
+    loader.resolve()
```

Comment:
I'd suggest adding at least one `assert` statement to this test. Perhaps assert that the value of `loader.schema.imports` contains the expected elements?